CHANGELOG.md

```diff
@@ -155,9 +150,6 @@ All notable changes to this project will be documented in this file.
 
 ### Feat
```

Comment:
We currently have:

### Feat

### Features

I think we want to get rid of the `### Feat` heading.